src/redisearch_rs/inverted_index_bencher/benches/numeric.rs

```diff
@@ -19,5 +19,14 @@ fn benchmark_numeric(c: &mut Criterion) {
     bencher.decoding(c);
 }
 
-criterion_group!(benches, benchmark_numeric);
+fn benchmark_freqs_only(c: &mut Criterion) {
+    let bencher = FreqsOnlyBencher::new();
+
+    bencher.encoding(c);
+    bencher.decoding(c);
+}
+
+// FIXME: should we move this to lib.rs? We can only have one criterion_main
+// and I'm not sure it's worth having a sub module for each type of benches.
+criterion_group!(benches, benchmark_numeric, benchmark_freqs_only);
```

Comment:
I was thinking we make a whole new file for each encoder / decoder under `benches` and then we make a section like this for each

https://github.com/RediSearch/RediSearch/blob/b5b49a96835391d68f1dac3d2c8f4939b118d0b3/src/redisearch_rs/inverted_index_bencher/Cargo.toml#L15-L17